microsite/blog/2020-09-08-announcing-tech-docs.md

```diff
@@ -6,6 +6,9 @@ authorURL: https://github.com/garyniemen
 
 Since we [open sourced Backstage](https://backstage.io/blog/2020/03/16/announcing-backstage), one of the most requested features has been for a technical documentation plugin. Well, good news. The first open source version of TechDocs is here. Now let’s start collaborating and making it better, together.
 
+<iframe width="780" height="440" src="https://www.youtube.com/embed/mOLCgdPw1iA" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen>
+</iframe>
+
```

Comment:
The video is linked further down in the blog (line 20). Could we embed the video there instead?

Reply:
We could but then it is show below the preview if you go to /blog

Reply:
ahh I see, nice, then we maybe can just delete the lines where it's linked out to?

Reply:
Feel free to update that Emma I’ll merge this one now.